samples/bluetooth/mesh_demo/boards/bbc_microbit.conf

```diff
@@ -22,3 +22,4 @@ CONFIG_BT_MESH_BEACON_ENABLED=n
 CONFIG_BT_MESH_LABEL_COUNT=1
 
 CONFIG_BT_MESH_SETTINGS_WORKQ=n
+CONFIG_BT_MESH_ACCESS_DELAYABLE_MSG=n
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/65485#discussion_r1413578447
It will be `y` by default after adapting the existing models to delayable message functionality. These changes will have to be done later. No reason to remove it.